addons/utm/models/utm.py

```diff
@@ -44,7 +44,7 @@ class UtmCampaign(models.Model):
         'utm.tag', 'utm_tag_rel',
         'tag_id', 'campaign_id', string='Tags')
 
-    is_website = fields.Boolean(default=False, help="Allows us to filter relevant Campaign")
+    is_relevant = fields.Boolean(default=False, help="Allows us to filter relevant Campaign")
```

Comment:
relevant is a strange naming ... but I don't have anything better. allow_management ? meh.